README.md

```diff
@@ -203,7 +203,7 @@ easy to fork and contribute any changes back upstream.
    Please make sure `eval "$(pyenv init -)"` is placed toward the end of the shell
    configuration file since it manipulates `PATH` during the initialization.
     ```sh
-    $ echo 'eval "$(pyenv init -)"' >> ~/.bash_profile
+    $ echo '[ -x ~/.pyenv/bin/pyenv ] && eval "$(pyenv init -)"' >> ~/.bash_profile
```

Comment:
It might be somewhere else..
You might want to include `$PYENV_ROOT` in the check and/or check for `PYENV_ROOT` being non-empty and use it in the `eval`.